source/clear-linux/get-started/virtual-machine-install/vmware-esxi.rst

```diff
@@ -1,80 +1,539 @@
+    #.  Click the :guilabel:`Power on` button.  
+    #.  Click on the icon representing the VM to bring it into view and maximize
+        its window.  
+
+       .. figure:: figures/vmware-esxi/vmware-esxi-11.png
+         :scale: 100 %
+         :alt: VMware ESXi - Navigator > Virtual Machines > Power on VM
+
+       Figure 11: VMware ESXi - Navigator > Virtual Machines > Power on VM
+
+#.  Follow the :ref:`install-on-target` guide to complete the installation of 
+    |CL|.
+#.  After the installation is complete, follow the |CL| instruction to reboot it.  
+    This will restart the installer again. 
+
+.. _label-vmware-esxi-reconfigure-vm-settings-to-boot-clear:
+
+Reconfigure the VM's settings to boot the newly-installed Clear Linux
+=====================================================================
+
+After |CL| has been installed using the installer ISO, it must be dettached so
+it won't run again.  Also, in order to boot the newly-installed |CL|, you must
+enable UEFI support. 
+
+#.  Power off the VM.
+
+    #.  Click the :guilabel:`Actions` button - located on the top-right corner 
+        of the VM's windows - and go to the :guilabel:`Power` setting and  
+        select the :guilabel:`Power off` option.  
+
+        .. figure:: figures/vmware-esxi/vmware-esxi-12.png
+          :scale: 100 %
+          :alt: VMware ESXi - Actions > Power off
+
+        Figure 12: VMware ESXi - Actions > Power off
+
+#.  Edit the VM settings.
+
+    #.  Click the :guilabel:`Actions` button again and select :guilabel:`Edit settings`.  
+
+        .. figure:: figures/vmware-esxi/vmware-esxi-13.png
+          :scale: 100 %
+          :alt: VMware ESXi - Actions > Edit settings
+
+        Figure 13: VMware ESXi - Actions > Edit settings
+
+#.  Disconnect the CD/DVD to stop it from booting the |CL| installer ISO again.
+    
+    #.  Click the :guilabel:`Virtual Hardware` button.
+    #.  For the :guilabel:`CD/DVD Drive 1` setting, uncheck the 
+        :guilabel:`Connect` checkbox.
+
+        .. figure:: figures/vmware-esxi/vmware-esxi-14.png
+          :scale: 100 %
+          :alt: VMware ESXi - Disconnect the CD/DVD drive
+
+        Figure 14: VMware ESXi - Disconnect the CD/DVD drive
+
+#.  |CL| needs UEFI support in order to boot.  Enable it.
+
+    #.  Click the :guilabel:`VM Options` button.
+    #.  Expand the :guilabel:`Boot Options` setting.
+    #.  For the :guilabel:`Firmware` setting, click the drop-down list to the right 
+        of it and select the :guilabel:`EFI` option.
+
+        .. figure:: figures/vmware-esxi/vmware-esxi-15.png
+          :scale: 100 %
+          :alt: VMware ESXi - Set boot firmware to EFI
+
+        Figure 15: VMware ESXi - Set boot firmware to EFI
+
+#.  Click the :guilabel:`Save` button.
+
+.. _label-vmware-esxi-power-on-vm-method-1:
+
+Power on the virtual machine
+============================
+
+After configuring the settings above, power on the virtual machine.  
```

Comment:
This section is nonsensical if you do not add the commands needed to do so.
Either remove it as a section or add the commands.